public/js/bundle.js

```diff
@@ -70,8 +70,27 @@ class TableModel {
   setValue(location, value) {
```

Comment:
Oops, your bundle.js file should not be checked into git or GitHub, since it's a big ugly mess of a file that production environments (Heroku) will generate for you anyway.

Please
1. delete the file from GitHub (all branches)
2. add bundle.js to your .gitignore file
3. run “git rm —cached public/js/bundle.js” in your terminal